packages/contracts-bedrock/src/L2/L2ToL2CrossDomainMessenger.sol

```diff
@@ -104,47 +119,49 @@ contract L2ToL2CrossDomainMessenger is IL2ToL2CrossDomainMessenger, ISemver, Tra
         if (_target == Predeploys.CROSS_L2_INBOX) revert MessageTargetCrossL2Inbox();
         if (_target == Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER) revert MessageTargetL2ToL2CrossDomainMessenger();
 
-        bytes memory data = abi.encodeCall(
-            L2ToL2CrossDomainMessenger.relayMessage,
-            (_destination, block.chainid, messageNonce(), msg.sender, _target, _message)
-        );
```

Comment:
since we are removing the `CrossL2Inbox` as entrypoint, has this been properly updated on the design doc, specs, etc?

Reply:
yep! the predeploys section for the messenger is updated in the [spec PR](https://github.com/ethereum-optimism/specs/pull/387/files#diff-c01dd3f111582c92427c67a2acfc13718ef88f5188424afaa1a8227968e5c41eR326).

Should there be a design doc for this?

Reply:
There seems to no be a design-doc for the CDM and no reference on how it should behave, there is no need to create one.